modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -68,6 +71,9 @@ components:
                         The blog post's author.
                     readOnly: true
                 customFields:
+                    # @review
+                    description:
+                        A list of the custom fields associated with the blog post.
```

Comment:
We can improve this type of comments to be more explicit. But I do not suggest you to change it, I want documentation to be involved on this. And we've hired 1 guy to help Spain (only) so he can tackle this. 

But here we should try to avoid comments like obvious javadocs: "A list of..." (and I've written the rest of them, so I'm the main culprit). Ideally this should bridge the needs of documentation of 2 users: the one that already knows liferay (custom fields that map to expando fields) and the ones that do not know liferay (custom fields allow you to define new fields to a Liferay entity).

Reply:
Sure. There are some obvious comments where I'm the first that I'm not fully understand how the REST entity maps with the Liferay entities and its usage... I'd preferred that everything had a description to be reviewed than let it without it